mobile_app.md

```diff
@@ -0,0 +1,16 @@
+# MiniAirQualityStation - Mobile App
+
+# Introduction
+
+The goal of the project is to create a mobile app that visualises information provided by sensors mounted on the Raspberry Pi based mini air quality station and configure it.
+
+# Specification
+
+## Mobile OS
+
+Google Android 8.0 (Oreo) or higher
+
+## Coding language and other technologies used
```

Comment:
Please include information about the IDE that we use (maybe in a separate paragraph). Also, consider if we decide to use WebView, it will be better to create separate documentation for the website, or to put it here.

Reply:
Added information regarding IDE. Keeping it open till decision has been made regarding web browser container.